src/pieces/commandManager.ts

```diff
@@ -14,6 +14,18 @@ async function register(bot: Client): Promise<void> {
 		bot.emit('error', error);
 	}
 
+	bot.on('messageCreate', async msg => {
+		if (msg.content.substring(0, 2).toLowerCase().includes('s;')) {
+			// eslint-disable-next-line max-len
+			msg.reply(`If you're trying to run a Sage command, we've moved over to using slash commands. If you're trying to enroll in a course, please use the dropdowns in <#${CHANNELS.ROLE_SELECT}> instead!`)
+				.then(reply => {
+					// delete reply after 10 seconds
+					setTimeout(() => { reply.delete(); }, 10000);
```

Comment:
Make this a constant
```suggestion
					setTimeout(() => { reply.delete(); }, DELETE_DELAY);
```